src/qt/test/apptests.cpp

```diff
@@ -82,6 +83,8 @@ void AppTests::appTests()
     // Reset global state to avoid interfering with later tests.
     AbortShutdown();
     UnloadBlockIndex();
+    ECC_Stop();
```

Comment:
In commit "qt: test: Create at most one testing setup" (999999bb11de41136b165e679b5828c4786a2f0f)

Why are ECC_Stop and DisconnectTestLogger needed here? Aren't they about to be called in the BasicTestingSetup destructor on the next line? It might be helpful to say something in a comment here, depending on the reason.

Reply:
Removed